### PR TITLE
Release note generate task

### PR DESCRIPTION
**Pull request by pocke**
Ref: https://skyarch.backlog.jp/view/NC-576

Usage
-------

```sh
bundle exec rake release:note_gen
```

### CODE REVIEW

File: lib/tasks/release.rake [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    if diff_files.find{|x| %r!^db/migrate/.+\.rb$!.match(x)}
+      puts 'bundle exec rake db:migrate'
+    end
+
+    if diff_files.include?('bower.json')
+      puts 'bower install'
+    end
+
+    npm_i = diff_files.include?('frontend/package.json')
+    gulp_ts = !!diff_files.find{|x| %r!^frontend/.+\.ts!}
```
Comment (skyarch-networks) [inline lib/tasks/release.rake]: __[rubocop]__
* Lint/UnusedBlockArgument: Unused block argument - `x`. You can omit the argument if you don't care about it. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

**Comment by pocke** [comment]
@Joeper214 please review and merge it :bow:

File: lib/tasks/release.rake (hunk 1 above)
Comment (pocke) [inline lib/tasks/release.rake]: :+1:

**Comment by Joeper214** [comment]
Does thos generate task creates a PR?

**Comment by pocke** [comment]
No, this task creates only template of PR. 
We can copy and modify output of `bundle exec rake release:note_gen`.

**Comment by pocke** [comment]
example output

    git fetch
    - elb-groups #153

    Features
    --------

    Bug fix
    --------

    Version up
    --------

    Others
    --------

    Update process
    --------

    ```sh
    cd YOUR_SKYHOPPER_DIRECTORY
    export RAILS_ENV=production  # This command isn't needed if you use develop environment.
    git pull
    git checkout <VERSION>
    cd frontend/
    gulp ts
    cd ..
    ```

**Comment by Joeper214** [comment]
:+1:

**Comment by pocke** [comment]
`git fetch` is noisy... I'll remove it

**Comment by Joeper214** [comment]
LGTM! :+1:

**Pull request merged**